tests/ecmp/test_fgnhg.py

```diff
@@ -198,6 +289,8 @@ def fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank
     configure_dut(duthost, "config interface shutdown " + dut_if_shutdown)
     time.sleep(30)
 
+   
+
```

Comment:
@kktheballer why do we need these extra spaces?

Reply:
addressed spacing comments